includes/class-sensei-analysis.php

```diff
@@ -190,6 +191,17 @@ public function analysis_admin_menu() {
 		}
 	}
 
+	/**
+	 * Enqueue JS scripts.
+	 *
+	 * @since 4.2.0
+	 */
+	public function enqueue_scripts() {
```

Comment:
Should this be `@access private`?

Reply:
Nice, only one docblock missed! Things are improving. 😄
Fixed in 819d824de15e54434e183fa36e5935b24a934c20.